qwandaq/src/main/docker/Dockerfile.jvm

```diff
@@ -75,9 +75,10 @@
 #   accessed directly. (example: "foo.example.com,bar.example.com")
 #
 ###
-FROM registry.access.redhat.com/ubi8/openjdk-11:1.11
+FROM registry.access.redhat.com/ubi8/openjdk-17:1.11
```

Comment:
Are we moving to Java17?

Reply:
Ahh this may have bled through from somewhere else. Not to my knowledge